dev-haskell/nats/nats-1.1.2-r1.ebuild

```diff
@@ -20,14 +20,18 @@ SLOT="0/${PV}"
 KEYWORDS="~amd64 ~x86"
 IUSE="+binary +hashable +template-haskell"
 
-RESTRICT=test # avoid circular dep: nats[hashable]->hashable[test]->quickcheck->nats
```

Comment:
Was removal of `RESTRICT` intentional?

Reply:
Yes I believe so, however the test cannot be run until the newer aeson is unmasked, so I'm going to pop this commit out of this PR and put it back into #1118